src/braidnet_epmd_server.erl

```diff
@@ -31,7 +31,7 @@
 -type nodenamehost() :: binary().
 %-------------------------------------------------------------------------------
 % We use this port range, realistically we will never spawn so many containers
--define(PORTS_RANGE, {50_000, 60_000}).
+-define(PORTS_RANGE, {50_100, 50_199}).
```

Comment:
There is no reason to modify this range right? for now we will only support one fixed 50100 port, but in theory we can configure ports for a huge fleet.